Branch-SDK/src/main/java/io/branch/referral/validators/BranchIntegrationModel.java

```diff
@@ -63,6 +64,17 @@ private void updateDeepLinkSchemes(Context context) {
         }
     }
 
+    //Function to get value from string resource
+    public String getStringResource(String stringWithIdValue){
+        if(stringWithIdValue.contains("resourceID")){
+            String resourceIdArr[] = stringWithIdValue.split(" ");
```

Comment:
C style declaration, replace with `String[] resourceIdArr`. In general please see what lint says and apply good judgement if you are ignoring it/add explanation.